src/Entities/Behaviors/HasCreatedDate.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Choredo\Entities\Behaviors;
+
+trait HasCreatedDate
+{
+    /**
+     * @ORM\Column(
+     *     type="datetime",
+     *     name="created",
+     *     nullable=false
+     * )
+     *
+     * @var \DateTime
+     */
+    private $createdDate;
+
+    /** @ORM\PrePersist */
+    public function updateCreatedDate()
```

Comment:
I know you got this from my code, but `updateCreatedDate` sounds weird. What are your thoughts? `setCreatedDate` sounds clearer/less confusing? NBD